static/app/views/replays/detail/network/details/sections.tsx

```diff
@@ -39,6 +39,12 @@ export type SectionProps = {
   startTimestampMs: number;
 };
 
+const config = {
```

Comment:
This config makes sense though! I think this is all you need

Reply:
It might make sense for this to be the default for the component and have the event details page pass in the custom config actually. But we can do that later